### PR TITLE
tools: fix environment-setup to work on aarch64

### PR DESCRIPTION
**Pull request by cjp256**
Signed-off-by: Chris Patterson <chris.patterson@canonical.com>

- [ ] Have you followed the [guidelines for contributing](https://github.com/snapcore/snapcraft/blob/master/CONTRIBUTING.md)?
- [ ] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [ ] Have you successfully run `./runtests.sh static`?
- [ ] Have you successfully run `./runtests.sh tests/unit`?

-----

### CODE REVIEW

File: tools/environment-setup.sh [hunk 1]
```diff
@@ -68,8 +71,8 @@ lxc exec snapcraft-dev -- sudo -iu ubuntu pip install \
 # Install the project for quick tests
 lxc exec snapcraft-dev -- sudo -iu ubuntu pip install --editable snapcraft
 
-# Install black to run static tests
-lxc exec snapcraft-dev -- snap install black --beta --devmode
+# Install black to run static tests, if available
```
Comment (sergiusens) [inline tools/environment-setup.sh:74]: Mind dropping this now due to https://snapcraft.io/black being on all architectures (well, except i386)?

Comment (cjp256) [reply tools/environment-setup.sh]: done!

File: tools/environment-setup.sh [hunk 2]
```diff
@@ -68,7 +71,7 @@ lxc exec snapcraft-dev -- sudo -iu ubuntu pip install \
 # Install the project for quick tests
 lxc exec snapcraft-dev -- sudo -iu ubuntu pip install --editable snapcraft
 
-# Install black to run static tests
+# Install black to run static tests.
 lxc exec snapcraft-dev -- snap install black --beta --devmode
```
Comment (sergiusens) [inline tools/environment-setup.sh:75]: you can probably remove --devmode here

Comment (sergiusens) [inline tools/environment-setup.sh:75]:
```suggestion
lxc exec snapcraft-dev -- snap install black --beta
```

**Pull request merged**